doc/doc-site/docs/move/changelog.md

```diff
@@ -20,6 +20,8 @@ Econia Move source code adheres to [Semantic Versioning] and [Keep a Changelog]
 
 ### Deprecated
 
+- [`market::MakerEvent`](https://github.com/econia-labs/econia/blob/v4.0.2-audited/src/move/econia/sources/market.move#L523) ([#321]).
```

Comment:
Feel like the list should go:

MakerEvent
maker_events
TakerEvent
taker_events
Orders

Reply:
@elliottdehn addressed per cd0b5ac7